packages/compass-query-bar/src/components/query-bar/query-bar.jsx

```diff
@@ -229,6 +229,35 @@ class QueryBar extends Component {
     const label = OPTION_DEFINITION[option].label || option;
     const placeholder = this.props[`${option}Placeholder`] || OPTION_DEFINITION[option].placeholder;
 
+    if (hasToggle) {
+      return (
+        <div
+          className={styles['query-option-toggle-row']}
+          style={{
+            display: 'flex',
+            flexDirection: 'row'
+          }}
+        >
+          <QueryOption
+            label={label}
+            autoPopulated={autoPopulated}
+            serverVersion={this.props.serverVersion}
+            hasToggle={hasToggle}
+            hasError={hasError}
+            key={`query-option-${id}`}
+            value={value}
+            actions={this.props.actions}
+            placeholder={placeholder}
+            link={OPTION_DEFINITION[option].link}
+            inputType={OPTION_DEFINITION[option].type}
+            onChange={this.onChange.bind(this, option)}
+            onApply={this.onApplyButtonClicked}
+            schemaFields={this.props.schemaFields}
+          />
```

Comment:
It's a bit annoying that this component is now just duplicated twice with the same exact set of option. It would be very easy to forget to keep them in sync when props are changed. Can we address this somehow?